apps.json

```diff
@@ -2401,6 +2401,12 @@
         ],
         "url": "https://github.com/YunoHost-apps/phpmyadmin_ynh"
     },
+    "phpservermon": {
+        "branch": "master",
+        "revision": "04c1ee06843ea4a7fd00b4eb6c381286f2562f1b",
```

Comment:
Note that we recommend using HEAD to avoid having to come back here to update the revision everytime you publish an update of the package (which generally ends up in people forgetting about it)